src/editing_tools/RedEyeTool.vala

```diff
@@ -45,9 +45,9 @@ public struct EditingTools.RedeyeInstance {
     }
 
     public static RedeyeInstance from_bounds_rect (Gdk.Rectangle rect) {
-        Gdk.Rectangle in_rect = rect;
+        var in_rect = rect;
```

Comment:
Hmm, it looks like we may actually not need the variable `in_rect`; it's essentially the same with the argument `rect` and only an  unnecessary copy happens heere. Maybe we can remove this and replace the occurance of `in_rect` to `rect` in  this method while we're here.